optaplanner-website-root/data/events.yml

```diff
@@ -14,6 +14,16 @@
 #
 # Order events by date descending
 
+- eventId: 20220414-devnexus-porter
+  eventOrganization: "DevNexus"
+  eventUrl: https://devnexus.org/
+  location: Atlanta, GA, USA
+  date: 2022-04-14
+  talks:
+    - title: "https://devnexus.org/"
```

Comment:
Thanks Jason for adding this :)

This field should hold the title of the talk (not an url as shown below), so I fixed that directly on main. I hop that's fine.

![image](https://user-images.githubusercontent.com/176880/162189027-916a4e7f-b3d4-4381-8616-3e90b90645b4.png)

Reply:
Oops, C&P error there :)